### ISSUE TITLE
Hiding icon and text when opened a dialog box

### ISSUE THREAD

**Issue by mmastriani** [psifidotos/applet-window-title#79]
The applet hides the icon and text when a dialog box is opened or when the application menu is opened. 

Thanks for the great work!

**Comment by psifidotos** [comment]
this is not fixable with plasma panels... use the applet in a Latte panel and you are ok...

**Comment by mmastriani** [comment]
I didn't know that is possible to use Latte as a panel. Thank you. Now it's near the perfection (the name keep unchanged when a dialog box is opened). But I realized that when I open the application menu, Window Title shows the Latte Icon and the text Latte. So, I decided to hide icons and created a substitution for "Latte" as "".

Now my "windows key" isn't showing my application menu. Is it possible to open it when in Latte?

Thank you so much!
